ambassador/Dockerfile

```diff
@@ -16,4 +16,4 @@ RUN apk update && \
 	apk add socat && \
 	rm -r /var/cache/
 
-CMD	env | grep _TCP= | sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/socat -t 100000000 TCP4-LISTEN:\1,fork,reuseaddr TCP4:\2:\3 \& wait/' | sh
+CMD	env | grep _TCP= | (sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/socat -t 100000000 TCP4-LISTEN:\1,fork,reuseaddr TCP4:\2:\3 \&/'; echo wait) | sh
```

Comment:
OK, `&&` is now. I have not re-tested it, though. But it should be fine I guess.